packages/core/src/messages/responses/cache-set-fetch.ts

```diff
@@ -64,6 +71,16 @@ class _Hit extends Response {
     return new Set(this.elements);
   }
 
+  /**
+   * Returns the data as an Array whose values are utf-8 strings, decoded from the underlying byte arrays.
+   * This accessor is provided because Arrays are sometimes easier to work with in TypeScript/JavaScript than Sets are.
+   * This is a convenience alias for {valueArrayString}.
+   * @returns {string[]}
+   */
+  public value(): string[] {
+    return this.valueArrayString();
+  }
```

Comment:
@allenheltondev this is probably the one that I was least certain which data type we should return for the new `value()` function. `Set` might be a more obvious choice here but I have always found `Set`s to be kind of a pain to work with in JS so I opted for an `Array` instead for the first pass.